cmake/appleclang-toolchain.cmake

```diff
@@ -16,8 +16,8 @@
 
 include_guard(GLOBAL)
 
-set(CMAKE_C_COMPILER clang)
-set(CMAKE_CXX_COMPILER clang++)
+set(CMAKE_C_COMPILER cc)
+set(CMAKE_CXX_COMPILER c++)
```

Comment:
Correct, but it's a stealth edit.

Commit message doesn't talk about editing the Apple Clang toolchain nor does the PR.

Reply:
Opened here: https://github.com/bemanproject/exemplar/pull/177